src/config/google.config.ts

```diff
@@ -0,0 +1,98 @@
+import passport from 'passport';
+import bcrypt from 'bcrypt';
+import { User } from '../models/index';
+import { Request } from 'express';
+import jwt from 'jsonwebtoken';
+import { Strategy as GoogleStrategy } from 'passport-google-oauth2';
+
+interface GoogleProfile {
+  id: string;
+  displayName: string;
+  name: { familyName: string; givenName: string };
+  email: { value: string; verified: boolean }[];
+  photos: { value: string }[];
+}
+
+const GOOGLE_CLIENT_ID = process.env.GOOGLE_CLIENT_ID!;
+const GOOGLE_CLIENT_SECRET = process.env.GOOGLE_CLIENT_SECRET!;
+const GOOGLE_CALLBACK_URL = process.env.GOOGLE_CALLBACK_URL!;
+const JWT_SECRET = process.env.JWT_SECRET;
+
```

Comment:
Reference the comment you left earlier. I've responded to it. From there you shall know how to modify this file with what to keep and remove from here to other files